aiohttp/client_exceptions.py

```diff
@@ -166,6 +167,14 @@ def __str__(self) -> str:
     __reduce__ = BaseException.__reduce__
 
 
+class ClientConnectorDNSError(ClientConnectorError):
```

Comment:
Not sure if this should be a connector error, though. Any reasoning?

Reply:
Is there much to discuss here? It looks to me like there's only 1 exception currently (there's only 1 place that raises this exception). It is also happening in the connector, so I don't really see any issue with it being a ConnectorError.

So, I think this change looks fine to me, and just needs the small bits of cleanup that has been asked for.